packages/swingset-runner/demo/resolveCircular/vat-bob.js

```diff
@@ -0,0 +1,43 @@
+import harden from '@agoric/harden';
+
+function makePR() {
+  let r;
+  const p = new Promise((resolve, _reject) => {
+    r = resolve;
+  });
+  return [p, r];
+}
+
+function build(_E, _log) {
+  let p1;
+  let r1;
+  let p2;
+  let r2;
+  return {
+    genPromise1() {
+      [p1, r1] = makePR();
+      return p1;
+    },
+    genPromise2() {
+      [p2, r2] = makePR();
+      return p2;
+    },
+    usePromises(pa, pb) {
```

Comment:
these demos are great.. do we have a test of cyclic resolution in swingset proper?

Reply:
I think we still need a unit test for this in `swingset`